recipes/viva-engage/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "viva-engage",
+  "name": "Viva Engage",
+  "alias": "yammer",
```

Comment:
this key is incorrect. please see other recipes that have aliases defined for eg aws

Reply:
My bad, fixed in 91849801a0238ec70c3c76a0d6d7b8b19cf0e9bd